docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Database Access with Oracle (Preview)
+description: How to configure Teleport Database Access with Oracle.
+---
+
+<Admonition type="warning" title="Preview">
+  Database Access for Oracle Server is currently in Preview mode.
+</Admonition>
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["cloud", "enterprise"]}
+  scopeOnly={true}
+  min="12.3"
+>
+  Database access for Oracle is available starting from Teleport `v12.3`.
+</Details>
+
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up Teleport to access your self-hosted Oracle server 18c or later.
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["enterprise"]}>
+![Teleport Database Access Self-hosted Oracle](../../../img/database-access/guides/oracle_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Oracle Cloud](../../../img/database-access/guides/oracle_selfhosted_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- Self-hosted Oracle server instance 18c or later.
+- The `sqlcl` [Oracle client](https://www.oracle.com/pl/database/sqldeveloper/technologies/sqlcl/) installed and added to your system's `PATH` environment variable or any GUI client that supports JDBC
+  Oracle thin client.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
```

Comment:
```suggestion
Follow the instructions below to install Teleport on the host where you will run the Teleport Database Service.
```
To match the partial, which includes introductory text that doesn't really work with a colon in the preceding paragraph.